Specs/createPackableSpecs.js

```diff
@@ -14,7 +14,8 @@ define([
 
         it(namePrefix + ' can pack', function() {
             var packedArray = [];
-            packable.pack(instance, packedArray);
+            packedArray = packable.pack(instance, packedArray);
+            expect(packedArray).toBeDefined();
```

Comment:
Rather than re-assigned packagedArray, create a new variable called `returnedArray` and do `expect(returnedAraray).toBe(packedArray)` in order to verify that's what is actually returned.